test/src/api/responderTest.php

```diff
@@ -7,6 +7,33 @@ function auth_session_exists() {
     return $dummyUserLoggedIn;
 }
 
+// Dieroller values which may change over time
+
+// RandomBM skills (used by all RandomBM types except RandomBMFixed)
+$RANDOMBM_SKILL = array(
```

Comment:
Sure, no problem - i didn't want to look for array indices inside each test, but doing one flip at the beginning seems totally reasonable.